jwst/extract_1d/ifu.py

```diff
@@ -451,6 +451,41 @@ def get_extract_parameters(ref_file, bkg_sigma_clip):
     return extract_params
 
 
+def _apply_bkg_sigma_clip(
+    bkg_data, temp_weightmap, bmask, bkg_sigma_clip, aperture_type, method, subpixels
+):
+    # pull out the data with coverage in IFU cube. We do not want to use
+    # the edge data that is zero to define the statistics on clipping
+    bkg_stat_data = bkg_data[temp_weightmap == 1]
+
+    # If there are good data, work out the statistics
+    if len(bkg_stat_data) > 0:
+        bkg_mean, _, bkg_stddev = stats.sigma_clipped_stats(
+            bkg_stat_data, sigma=bkg_sigma_clip, maxiters=5
```

Comment:
At least explain why it is hardcoded at 5 to make it less magical? Thanks!

Reply:
5 is the default.  We can just stop specifying it.

Reply:
This would work too, yes. 
```suggestion
            bkg_stat_data, sigma=bkg_sigma_clip
```